src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -199,6 +217,24 @@ protected void copyVideos(final String containerId) throws IOException, DockerEx
         LOGGER.log(Level.INFO, "{0} Video files copies to: {1}", new Object[]{getNodeIpAndPort(), localPath});
     }
 
+    private static void readEnvVarForVideoRecording(DockerSeleniumRemoteProxy proxy) {
+        if (environment.getEnvVariable(ZALENIUM_VIDEO_RECORDING_ENABLED) != null) {
+            try {
+                boolean videoEnabled = Boolean.parseBoolean(environment.getEnvVariable(ZALENIUM_VIDEO_RECORDING_ENABLED));
+                setVideoRecordingEnabled(videoEnabled);
+            } catch (Exception e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)